piker/ui/_display.py

```diff
@@ -416,10 +420,8 @@ def graphics_update_cycle(
             )
             or trigger_all
         ):
-            # TODO: we should track and compute whether the last
-            # pixel in a curve should show new data based on uppx
-            # and then iff update curves and shift?
             chart.increment_view(steps=i_diff)
+            # chart.increment_view(steps=i_diff + round(append_diff - uppx))
```

Comment:
left this is in because to me the logic makes sense but for whatever reason it doesn't seem to actually be correct based on testing.

Ideally we can get to where there's never an error margin on shifts when optimizing for uppx scaling.